app/assess/data.py

```diff
@@ -86,20 +96,28 @@ def get_rounds(fund_id: str) -> Union[Fund, List]:
     return rounds
 
 
-def get_round_with_applications(
-    fund_id: str, round_id: str
-) -> Union[Round, None]:
+def get_round(fund_id: str, round_id: str) -> Union[Round, None]:
     round_endpoint = Config.FUND_STORE_API_HOST + Config.ROUND_ENDPOINT.format(
         fund_id=fund_id, round_id=round_id
     )
     round_response = get_data(round_endpoint)
-    if round_response and "round_id" in round_response:
+    if round_response and "assessment_deadline" in round_response:
+        round = Round.from_dict(round_response)
+        return round
+    return None
+
+
+def get_round_with_applications(
+    fund_id: str, round_id: str
+) -> Union[Round, None]:
+    round_response = get_round(fund_id, round_id)
+    if round_response:
         fund_round = Round.from_json(round_response)
         applications_response = call_search_applications(
             {
                 "fund_id": fund_id,
                 "datetime_start": fund_round.opens,
-                "datetime_end": fund_round.deadline,
+                "datetime_end": fund_round.short_name,
```

Comment:
I wonder if 'short_name' and 'datetime_end'  is potentially confusing here?

Reply:
ah that's a refactor in pycharm gone wrong... good spot :)